src/aohara/tinkertime/crawlers/pageLoaders/JsonLoader.java

```diff
@@ -3,12 +3,16 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.Reader;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
```

Comment:
Unused imports are Authenticator, PasswordAuthentication, UrlConnection, and JsonElement